doc/tapyrus/getting_started.md

```diff
@@ -12,7 +12,6 @@ Tapyrus network has different kinds of nodes, this is a minimal list at the time
 | Tapyrus Core  | min 1, unbounded  | Tapyrus full node implementation |
 | Tapyrus Seeder | min 1, unbounded  | DNS seeder for configuring Tapyrus network |
 
-
 ### Tapyrus network parameters
 Following parameters are unique to each tapyrus network
 
```

Comment:
Please add description of the startup method and environment specifying the networkid.